vnext/Shared/IWebSocketResource.h

```diff
@@ -86,7 +86,7 @@ struct IWebSocketResource {
   static std::shared_ptr<IWebSocketResource>
   Make(const std::string &url, bool legacyImplementation = false, bool acceptSelfSigned = false);
```

Comment:
(nit) can we reformat this method to:
static std::shared_ptr<IWebSocketResource> Make(
    const std::string &url,
    bool legacyImplementation = false,
    bool acceptSelfSigned = false);

Reply:
Not sure if clang format will take it, but will also remove one, or maybe both boolean args now that "feature gates" are supported.